test/verify/check-example

```diff
@@ -37,6 +37,15 @@ class TestExample(MachineCase):
         self.login_and_go("/system")
 
 
+class TestPersistentExample(PersistentMachineCase):
+
+    def testOne(self):
+        self.assertEqual(self.machine.execute("whoami").strip(), "root")
```

Comment:
It just occurred to me -- `testOne` should create a file in /run on the VM, and testTwo should assert its existence. To prove that they both run in the same VM.

Reply:
that's a good idea! done